src/libraries/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -3546,17 +3545,16 @@ public partial class XmlDataDocument : System.Xml.XmlDocument
         public XmlDataDocument(System.Data.DataSet dataset) { }
         public System.Data.DataSet DataSet { get { throw null; } }
         public override System.Xml.XmlNode CloneNode(bool deep) { throw null; }
-        public override System.Xml.XmlElement CreateElement(string prefix, string localName, string namespaceURI) { throw null; }
+        public override System.Xml.XmlElement CreateElement(string? prefix, string localName, string? namespaceURI) { throw null; }
         public override System.Xml.XmlEntityReference CreateEntityReference(string name) { throw null; }
-        protected override System.Xml.XPath.XPathNavigator CreateNavigator(System.Xml.XmlNode node) { throw null; }
+        protected override System.Xml.XPath.XPathNavigator? CreateNavigator(System.Xml.XmlNode node) { throw null; }
         public override System.Xml.XmlElement GetElementById(string elemId) { throw null; }
```

Comment:
```suggestion
        public override System.Xml.XmlElement? GetElementById(string elemId) { throw null; }
```
Same [comment](https://github.com/dotnet/runtime/pull/41541/files/c19f67efc87c2882efa7ed07ed9afc02483ec51f#r479589562) as in src, this returns nullable in base.

Reply:
Technically, it doesn't return anything because it throws 😉 

Will fix!

Reply:
Is this fixed? Seems like is fixed in the implementation but not in the ref:
https://github.com/dotnet/runtime/blob/c4cacc0d8611fdbe8eb76dccef8e6e6961cd685f/src/libraries/System.Data.Common/src/System/Xml/XmlDataDocument.cs#L3005

Reply:
Fixed in https://github.com/dotnet/runtime/pull/41541/commits/9ad4719e591029ea83e290ec66144c3396696f38.